README.md

```diff
@@ -1,3 +1,40 @@
-# FixMe
-
+# FixMe.jl – A Beginner Friendly Platform for Making Your Own Package
 [![Build Status](https://github.com/CasBex/FixMe.jl/actions/workflows/CI.yml/badge.svg?branch=main)](https://github.com/CasBex/FixMe.jl/actions/workflows/CI.yml?query=branch%3Amain)
+
+FixMe.jl is a package for aspiring Julia package developers/contributors to learn about the development process.
+We provide a low stakes, low barrier-to-entry environment where you can learn the ins and outs of how to file issues, merge bugfixes/features to your favorite package, and how to get started developing your own packages.
+Pair up with a mentor and get started on the assignments below.
+
+The code in this package has various issues: it sometimes returns incorrect results or errors, some functions are very inefficient, documentation is poor... The assignments will help you get started fixing them and merging your fixes through the proper channels.
+
+**This package is not intended for productive use!**
+
+## For Mentors
+
+In these assignments you will act as package maintainer.
+Fork this repo and ask your mentee to file issues and open pull-requests to your fork.
+We ask you to review the pull-requests of your mentee and maybe give helpful suggestions to improve their Julia skills. 
+Please also help your mentee out when they have problems with git, setting up documentation with Documenter...
+
+## For Mentees
+In these assignments you will identify bugs/problems in the FixMe.jl package.
+You will learn how to make issues on Github, how to fix the problems in FixMe.jl and how to get your fix merged into the package.
+
+We aim to provide a safe space to sharpen your Julia skills. 
+You can make plenty of mistakes here without introducing bugs into other peoples' workflows.
+You don't need to worry about your issue/pull-request hurting the feelings of the package developers who worked super hard on their amazing package.
+This is a valuable opportunity to learn how to avoid all of this in practice though.
+
+To get you started, some helpful resources are
+- Git for beginners **TODO add link** (I'm open to suggestions)
```

Comment:
I think https://learngitbranching.js.org/ is quite popular. I like it because it's interactive and gives you a visualization of what's going on

Reply:
That seems like a great resource. Added it